recipe/one_more.patch

```diff
@@ -0,0 +1,35 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 1e6985d4..091b66c4 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -51,11 +51,7 @@ find_package(Threads REQUIRED)
+ 
+ 
+ if(MSVC)
+-    # Disable autolinking on MSVC.
+-    add_definitions(-DBOOST_ALL_NO_LIB)
```

Comment:
I feel like this is no longer necessary:
https://github.com/boostorg/boost_install/issues/54